### PR TITLE
change global time animation

### PR DESCRIPTION
**Pull request by DzesikaTkaczyk**

### CODE REVIEW

File: src/sass/_variables.scss [hunk 1]
```diff
@@ -9,3 +9,11 @@ $primary: #d58e32;
 $text-color: #2a2a2a;
 
 $form-border-color: #292929;
+
+$background-color-animation-time: 250ms;
```
Comment (matwjablonski) [inline src/sass/_variables.scss]: what with .3s and .7s - these values are in previous PRs

Comment (DzesikaTkaczyk) [reply src/sass/_variables.scss]: mam uwzględniać niezatwierdzone zmiany innych osób w swoim kodzie?

Comment (matwjablonski) [reply src/sass/_variables.scss]: please talk with your team about these values - and use same for everything, don't add another animation time

**Comment by matwjablonski** [comment]
please resolve conflicts

**Comment by danielmark0116** [comment]
As for the conflicts -> in variables.scss the breakpoints variables are used in _breakpoints.scss (mixins for phone/tablet/desktop media) so they should stay.

shortInterval i veryShortInterval are used in fade.scss for transition mixin (fade and fade-long, both being in use). We discussed it with @DzesikaTkaczyk and either:

1) we delete them and leave $background-color-animation-time. but then inside fade.scss someone has to change $veryShortInterval into $background-color... and $shortInterval into $background... * 2

2) or leave both?

what do you think ? :)

**Comment by matwjablonski** [comment]
change unnecessary values to these two you are leaving in project in this PR :)

**Comment by DzesikaTkaczyk** [comment]
dodałam hovery i moje czasy animacji na całej stronie 
wygląda jak powinno

File: src/sass/components/_footer.scss [hunk 2]
```diff
@@ -19,9 +19,12 @@ footer {
             font-size: 14px;
             line-height: 30px;
             color: rgb(169, 169, 169);
+            @extend %hover-animation-time;
 
-            &:hover {
-              color: #ffffff;
+            @include hover-only {
+              &:hover {
+                color: #ffffff;
```
Comment (matwjablonski) [inline src/sass/components/_footer.scss]: color from variable, sorry ;)

Comment (DzesikaTkaczyk) [reply src/sass/components/_footer.scss]: Ok, poprawione. Nie zauważyłam tego bo właściwie nie pracowałam na tym pliku, przeszukiwałam tylko pod kątem hoverów

**Pull request merged**